src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/IPOContextExtension.cs

```diff
@@ -65,12 +77,15 @@ private static void SeedCurrentUserAsPerson(IPOContext dbContext, ICurrentUserPr
 
         private static Invitation SeedMdpInvitation(IPOContext dbContext, string plant)
```

Comment:
take project as arg, and remove added lines using ProjectRepository. Same for SeedDpInvitation below